app/data/models/returns.js

```diff
@@ -0,0 +1,37 @@
+module.exports = (sequelize, DataTypes) => {
+  const returns = sequelize.define('returns', {
```

Comment:
Still suggest changing this bit `sequelize.define('returns', {` to be `return` and any calls to it to `db.return`.  Just because it's representing a single model, not a collection

Reply:
This should be resolved already :)

Reply:
It still says `return`?